2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_setup.py:_flush():69] setting env: {}
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_setup.py:_flush():69] setting login settings: {}
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_init.py:_log_setup():319] Logging user logs to /home/dmivilensky/Diffusion-LM/wandb/run-20221219_152753-refactoring/logs/debug.log
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_init.py:_log_setup():320] Logging internal logs to /home/dmivilensky/Diffusion-LM/wandb/run-20221219_152753-refactoring/logs/debug-internal.log
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_init.py:init():352] calling init triggers
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_init.py:init():357] wandb.init called with sweep_config: {}
config: {}
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [wandb_init.py:init():401] starting backend
2022-12-19 15:27:53,649 INFO    MainThread:3453869 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-12-19 15:27:53,692 INFO    MainThread:3453869 [backend.py:ensure_launched():123] starting backend process...
2022-12-19 15:27:53,738 INFO    MainThread:3453869 [backend.py:ensure_launched():127] started backend process with pid: 3457831
2022-12-19 15:27:53,740 INFO    MainThread:3453869 [wandb_init.py:init():406] backend started and connected
2022-12-19 15:27:53,741 INFO    MainThread:3453869 [wandb_init.py:init():446] updated telemetry
2022-12-19 15:27:53,742 INFO    MainThread:3453869 [wandb_init.py:init():465] communicating current version
2022-12-19 15:27:54,461 INFO    MainThread:3453869 [wandb_init.py:init():470] got version response upgrade_message: "wandb version 0.13.7 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-12-19 15:27:54,461 INFO    MainThread:3453869 [wandb_init.py:init():478] communicating run to backend with 30 second timeout
2022-12-19 15:28:03,646 WARNING MainThread:3453869 [wandb_init.py:init():748] interrupted
Traceback (most recent call last):
  File "/home/dmivilensky/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 728, in init
    run = wi.init()
  File "/home/dmivilensky/.local/lib/python3.8/site-packages/wandb/sdk/wandb_init.py", line 479, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "/home/dmivilensky/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 640, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/dmivilensky/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 628, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/dmivilensky/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 525, in _communicate
    f = future.get(timeout)
  File "/home/dmivilensky/.local/lib/python3.8/site-packages/wandb/sdk/interface/interface.py", line 81, in get
    is_set = self._object_ready.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 558, in wait
    signaled = self._cond.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 306, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
